test/hedera/specs/LiveContract.Hedera.spec.ts

```diff
@@ -1,97 +1,37 @@
 import {
   describe, expect, it
 } from '@jest/globals';
+import { Hbar } from "@hashgraph/sdk";
 
-import { Hbar, TokenType } from "@hashgraph/sdk";
-
-import { Account, KeyType } from '../../../lib/static/create/Account';
+import { defaultEd25519AccountFeatures, defaultFungibleTokenFeatures } from "../../constants";
+import { Account } from '../../../lib/static/create/Account';
 import { ApiSession } from '../../../lib/ApiSession';
 import { Contract } from '../../../lib/static/upload/Contract';
 import { Token } from '../../../lib/static/create/Token';
-import { defaultNonFungibleTokenSpecs } from "../../constants";
 import { read } from "../../utils";
 
 describe('LiveContract.Hedera', () => {
+
   it("given a fungible, live, token, minting over a precompiled contract-service bridge should be permitted", async () => {
     const contract = await Contract.newFrom({ code: read({ contract: 'HelloWorldMint', relativeTo: 'hedera' }), ignoreWarnings: true });
-    const token = new Token({ name: "PLM", symbol: "$", type: TokenType.FungibleCommon, initialSupply: 1000 });
+    const token = new Token({... defaultFungibleTokenFeatures, ...{initialSupply: 1000}});
```

Comment:
`const token = new Token({... defaultFungibleTokenFeatures, initialSupply: 1000});`

No need to spread the second object.